spec/sessions/sign_up_spec.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+require_relative '../helpers/spec_helper'
+require_relative '../helpers/auth_helper'
+require_relative '../helpers/logs_helper'
+require_relative '../../app/controllers/sessions_controller'
+require_relative '../../app/controllers/records_controller'
+require 'yaml'
+
+data = YAML.load_file('data.yml')
+
+RSpec.describe '[Sessions API]' do
+  include AuthHelper
+  include LoggerHelper
+
+  def app
+    Rack::Builder.new do
+      run ApplicationController
+      use RecordsController
+      use SessionsController
+      use UsersController
+    end.to_app
+  end
+
+  describe 'GET /sign_up_form' do
```

Comment:
We don't write spec with a lot of `describe` as usual. You should have only one `describe` in row 11 and that's all. 
Here you can use `context` inside `context`. 
As usual, we start context with word `when` and `it` with word `verify` or `verifies`.
Ex:
```
context 'when on Sign Up page' do
  context 'when login with invalid data' do
    it 'verify error message is displayed' do
      #test
    end
  end
end
```
 Kinda like this
 To have in logs logical sentences about what was tested

Reply:
Thank you for the comments! I have a question regarding this one. Should I define if it's GET or POST request in context to?

Reply:
done ✅